js_modules/dagit/packages/core/src/ui/Filters/useFilter.tsx

```diff
@@ -35,17 +28,19 @@ export const FilterTag = ({
   iconName: IconName;
   onRemove: () => void;
 }) => (
-  <BaseTag
-    icon={<Icon name={iconName} color={Colors.Link} />}
-    rightIcon={
-      <div onClick={onRemove} style={{cursor: 'pointer'}} tabIndex={0}>
-        <Icon name="close" color={Colors.Link} />
-      </div>
-    }
-    label={label}
-    fillColor={Colors.Blue50}
-    textColor={Colors.Link}
-  />
+  <div style={{display: 'inline-block', height: '24px'}}>
```

Comment:
If this is to avoid the vertical stretching on the tag, it might be sufficient just to have an unstyled `div` wrapper, since then the `BaseTag` won't be affected by the flex properties of the container.

Reply:
ah let me try that